### PR TITLE
Upgrades

### PR DESCRIPTION
**Pull request by jeffdechambeau**
- [x] **Get excluded calendars + events config from local json**
- [x] **Don't show nag if Focus Bear app is installed**
- [x] **Save passed events list to avoid duplicate notifs** 
- [x] **Show icon in dialog & migrated to transparent PNG, attached** 
Only applies to final dialog, lmk if it should apply to all, but that didn't track with the 00:00 time left in the image. Assumes the png is in the user's Application Support/com.focusbear.latenomore
- [x] **Only show Focus Bear nag once every 7 days**
Uses `calendar date % 7 == 0` -- which will cause the event to shift around as the months change, which strikes me as a good thing since it adds a bit more randomness to it while not happening more frequently than every 7th day
- [x] **Add ga utm params to URL Focus Bear homepage url**

icon

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,14 +15,37 @@ const {
...
+
+const {
+    setCalsToExclude,
+    setEventsToExclude,
+} = require('./src/applescript/calendar.js')
+
+const { setNagState } = require('./src/intention.js')
+
+async function setSettings() {
+    const obj = await readSettings()
```
Comment (jeznag) [inline index.js:31]: is this needed?

Comment (jeffdechambeau) [reply index.js]: If you mean the obj declaration, nope, that was leftover from testing. Removed in the latest commit.

File: index.js [hunk 2]
```diff
@@ -15,14 +15,36 @@ const {
...
     console.log('Late No More Online..')
+    await setSettings()
+
     if (IS_TESTING) await addTestEvents()
     else syncCalendarsToUpcoming()
     checkUpcomingForMeetings()
 
     setInterval(checkUpcomingForMeetings, quickInterval)
     setInterval(syncCalendarsToUpcoming, slowInterval)
+    setInterval(setNagState, 1000 * 60 * 60 * 24)
```
Comment (jeznag) [inline index.js:47]: can we have a constant for this? ONE_DAY_IN_SECONDS

Comment (jeffdechambeau) [reply index.js]: Done in latest commit.

**Pull request merged**